supplementary_materials/notebooks/fastdraft-deepseek/fastdraft_deepseek.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    def streamer(subword):
</pre><p>defined twice, probably one of them can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2717/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done